ts/src/binance.ts

```diff
@@ -4218,6 +4218,9 @@ export default class binance extends Exchange {
         const price = this.safeString (params, 'price');
         const until = this.safeInteger (params, 'until');
         params = this.omit (params, [ 'price', 'until' ]);
+        if (since !== undefined && until !== undefined && limit === undefined) {
```

Comment:
@olegtaranenko If since and until are provided, doesn't that create a limit already? 
Can you please give an example where this would be beneficial?

Reply:
@carlosmiei correct. Binance do not document, that limit is required to overcome default limit size (500), even startTime and endTime are both set. You can view the comment above my change in the code.
```
        // binance docs say that the default limit 500, max 1500 for futures, max 1000 for spot markets
        // the reality is that the time range wider than 500 candles won't work right
```
Example? I'm writing fetching and caching locally with arbitrary access by time. Then to me is important to fill holes in history, where I know tails and heads of the missing candles' ranges

Reply:
forget to add, that providing of all three parameters is safe - result will not beyond start and end times

Reply:
@olegtaranenko thanks for clarifying it, will add a test and merge it!